apps/documentation/src/routes/documentation/contributing.mdx

```diff
@@ -14,35 +14,35 @@
 
 ## TL;DR
 
-The project is massive - if you like it do consider to contribute!
+The project is massive - if you like it, do consider contributing!
 
 ## Getting started
 
-The `tuono` project can mostly be split by the following sub-domains:
+The `tuono` project can mostly be split into the following subdomains:
 
 - The CLI
-- The rust backend
-- The react frontend
+- The Rust backend
+- The React frontend
 - The documentation website (which is written with tuono 🚀)
 
-To check what are the knowledge requirement for each domain check the
+To check what the knowledge requirements are for each domain, check the
 [requirements](#requirements) section below.
 
-Currently I'm keeping a private dashboard to prioritize new features and bug fixes but if you
-want to propose something please open a new issue on github or reach me using
+Currently, I'm keeping a private dashboard to prioritize new features and bug fixes, but if you
+want to propose something, please open a new issue on Github or reach out to me using
 my email address [valerioageno@yahoo.it](mailto:valerioageno@yahoo.it). I'm also available
-on twitter (X) DMs `@valerioageno`, [linkedin](https://www.linkedin.com/in/valerioageno)
+on Twitter (X) DMs `@valerioageno`, [Linkedin](https://www.linkedin.com/in/valerioageno)
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The name of this social business platform is spelled with a capital “I”. (LINKEDIN[1])
Suggestions: `LinkedIn`
URL: https://en.wikipedia.org/wiki/LinkedIn 
Rule: https://community.languagetool.org/rule/show/LINKEDIN?lang=en-US&subId=1
Category: CASING
<!-- __reviewdog__:ChBmMDQyOTI0NTA2Nzg3MjhjEgxMYW5ndWFnZVRvb2w= -->